vNext/Test.cmd

```diff
@@ -10,18 +10,27 @@ set CMDHOME=%CMDHOME:~0,-1%
 pushd "%CMDHOME%"
 @cd
 
-SET OutDir=%CMDHOME%\Binaries\%CONFIGURATION%
+SET TestResultDir=%CMDHOME%\Binaries\%CONFIGURATION%\TestResults
```

Comment:
This directory is a little odd right now, since we no longer produce binaries. Nevertheless, let's not fix it right away, since VSO and Jenkins will probably be confused if we change it